manifests/1.4.0/opensearch-dashboards-1.4.0.yml

```diff
@@ -5,7 +5,7 @@ build:
   version: 1.4.0
 ci:
   image:
-    name: opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211028
+    name: opensearchstaging/ci-runner:ci-runner-centos7-opensearch-dashboards-build-v2
```

Comment:
should resolve: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2685

Reply:
Hi @kavilla isnt 1.x will lock to node 10.24.1?

Reply:
This image hasn't been updated for 1.4 but all the other builds on 1.x use this image. For example, https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.3.9/opensearch-dashboards-1.3.9.yml#LL8C59-L8C87